omisego-sdk/src/main/java/co/omisego/omisego/operation/Listenable.kt

```diff
@@ -34,6 +35,53 @@ interface Listenable {
     }
 }
 
+@Deprecated(
+    level = DeprecationLevel.ERROR,
+    message = "Use startListeningEvents(SocketClientContract.Client, Map<String, Any>, TransactionRequestTopicListener) instead."
+)
+fun TransactionRequest.startListeningEvents(
+    client: SocketClientContract.Client,
+    payload: Map<String, Any> = mapOf(),
+    listener: SocketCustomEventListener.TransactionRequestListener
+) {
+    with(client) {
+        listener.strategy = FilterStrategy.Topic(socketTopic)
+        addCustomEventListener(listener)
+        joinChannel(socketTopic, payload)
+    }
+}
+
+@Deprecated(
+    level = DeprecationLevel.ERROR,
+    message = "Use startListeningEvents(SocketClientContract.Client, Map<String, Any>, TransactionConsumptionTopicListener) instead."
+)
+fun TransactionConsumption.startListeningEvents(
+    client: SocketClientContract.Client,
+    payload: Map<String, Any> = mapOf(),
+    listener: SocketCustomEventListener.TransactionConsumptionListener
+) {
+    with(client) {
+        addCustomEventListener(listener)
```

Comment:
Not changing the strategy on this one? 🤔

Reply:
Will change it but this one will be deprecated as a compile error anyway 😄